lib/HighlightsFeed.jsm

```diff
@@ -280,10 +252,10 @@ this.HighlightsFeed = class HighlightsFeed {
         this.fetchHighlights({broadcast: true});
         break;
       case at.DELETE_FROM_POCKET:
-        this.deleteFromPocket(action.data.pocket_id);
+        this.fetchHighlights({broadcast: true});
         break;
       case at.ARCHIVE_FROM_POCKET:
-        this.archiveFromPocket(action.data.pocket_id);
+        this.fetchHighlights({broadcast: true});
```

Comment:
I think it's confusing why a `ARCHIVE_FROM_POCKET` case fires off a `fetchHIghlights`